hivemind/p2p/p2p_daemon.py

```diff
@@ -90,6 +92,7 @@ async def create(
         use_auto_relay: bool = False,
         relay_hop_limit: int = 0,
         startup_timeout: float = 15,
+        idle_timeout: float = 0,
```

Comment:
```suggestion
        idle_timeout: float = 30,
```

Let's enable this by default.